src/App.tsx

```diff
@@ -61,7 +65,7 @@ export const UnconnectedApp: React.FC<AppProps> = ({
               path="/"
               exact={true}
               render={() => {
-                setSubtitle(VIEWS.CASES_TO_WORK.TITLE);
+                setSubtitleAndClearCases(VIEWS.CASES_TO_WORK.TITLE);
```

Comment:
Seems a little messy to combine set Subtitle and clear cases into a single function. Although looking at this I think there is a bigger issue. It looks like this is setting state with in the render cycle.

Maybe not in this PR but I think we should refactor this so that:

- `clearCase()` is called when the case list component (snooze/active) is mounted
- Setting the subtitle seems like something the belongs with in the Layout component
- I think we should also avoid using the render prop on the route and pass the CaseList in as a component.
- refactor Layout to be connected with Redux so we don't need a render function for that either

Reply:
By the way, I have a subsequent PR waiting in draft that accomplishes what you're suggesting: https://github.com/usds/case-issue-navigator/pull/101.